src/js/common/schemaform/helpers.js

```diff
@@ -107,13 +124,21 @@ export function formatISOPartialDate({ month, day, year }) {
   return undefined;
 }
 
+export function formatReviewDate(dateString) {
+  if (dateString) {
+    const [year, month, day] = dateString.split('-', 3);
+    return `${formatDayMonth(month)}/${formatDayMonth(day)}/${formatYear(year)}`;
+  }
+
+  return undefined;
+}
 export function parseISODate(dateString) {
   if (dateString) {
     const [year, month, day] = dateString.split('-', 3);
 
     return {
       month: month === 'XX' ? '' : Number(month),
-      day: day === 'XX' ? '' : Number(day),
+      day: day === 'XX' ? '' : Number(day).toString(),
```

Comment:
The way our current date component works (which I copied), it requires the month to be a number and the day to be a string.